resources/heroku-docker-start.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+a2dismod mpm_event &&
```

Comment:
I don't think `&&` is necessary here.
```suggestion
a2dismod mpm_event
```

Reply:
yup.thats right